### ISSUE TITLE
Job model change

### ISSUE THREAD

**Issue by pfurio** [opencb/opencga#1548]
In order to support multistudy jobs (ticket), we need to apply a small change to the Job data model. We will add a new "study" field object that contains the following structure:
```
study: {
   id: "user@project:study1",
   others: ["user@project:study2", "user@project:study3"]
}
```
Assuming user 'user' has a project 'project' that contains three different studies 'study1', 'study2' and 'study3', and that a user has sent a project centric job, the structure will be automatically fulfilled in the way showed above.

Basically, study.id will always contain the main study any file result will be associated to. The user needs to decide which this main study will be. After that, the daemon will automatically complete the study.others field information.

Additionally, because of this new structure, we will remove the 'studyUuid' field from the job data model as will no longer be necessary. 

The final list of tasks are:

- [ ] Add the new _study_ field and remove _studyUuid_.
- [ ] Add a migration script.
